go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/fsouza/fake-gcs-server v1.40.3
 	github.com/googleapis/gax-go/v2 v2.11.0
 	github.com/jacobsa/daemonize v0.0.0-20160101105449-e460293e890f
-	github.com/jacobsa/fuse v0.0.0-20231003132804-d0f3daf365c3
+	github.com/jacobsa/fuse v0.0.0-20230810134708-ab21db1af836
```

Comment:
I didn't understand, why do we need to degrade the jacobsa/fuse version?

Reply:
Dependabot raised PR with this fuse version, I have fixed that.